parser/src/ast/display.rs

```diff
@@ -111,6 +111,7 @@ impl fmt::Display for DisplayStatement<'_> {
                 write!(f, "enf {}", expr)
             }
             Statement::Expr(ref expr) => write!(f, "return {}", expr),
+            Statement::BusEnforce(ref expr) => write!(f, "bus_enf {}", expr),
```

Comment:
For display, this should probably be `enf` as well, just like `EnforceAll`, no? There is no `bus_enf` keyword in any case. For `Debug` purposes though, obviously you'd want to distinguish the two, but `Display` is usually used for pretty printing, so it should probably follow the actual AirScript syntax as much as possible.